### PR TITLE
Uploading files in the styling branch of the Bokk-Store-Redux project

### PR DESCRIPTION
**Pull request by siddghosh108**
PR Description

Added any missing JSX elements to my project (without adding extra functionality)
My final design matches Zeplin's design (this includes all elements + the same styling)
I used the CSS rules detailed in Zeplin to style the website.

The UI looks as follows after npm start
image

Please give your review
Thanks

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
+            name: 'CATEGORIES',
+          },
+        ].map((link) => (
+          <li key={link.id}>
+            <NavLink className="nav-link" to={link.path}>
+              {link.name}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
```
Comment (danmainah) [inline src/components/Navbar.js:33]: - [ ] @siddghosh108 Great work styling your app, however I noticed a slight difference in styling between your navbar items and the given design, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image  | image

File: src/components/BookComp.js [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
+            max="100"
+            style={{
+              height: 0,
+              width: 0,
+            }}
+          >
+            75%
+          </progress>
+        </div>
+        <div className="book-container2">
```
Comment (danmainah) [inline src/components/BookComp.js:42]: - [ ] @siddghosh108 Great work styling the app, however,  the thickness is this section is bigger than in the design, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image  | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @siddghosh108 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Added styling to the page and ensured it matches the Zeplin design given.
✔️  Ensured all elements from the design are added.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Navbar.js (hunk 1 above)
Comment (siddghosh108) [reply src/components/Navbar.js:33]: Now the font is thin and sleek just like the way you wanted

File: src/components/BookComp.js (hunk 2 above)
Comment (siddghosh108) [reply src/components/BookComp.js:42]: I have decreased thethickness of the progressbar the way you wanted. Please see below:

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @siddghosh108 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
 

## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**